tools/integration_tests/util/creds_tests/testdata/revoke_permission_and_delete_service_account_and_creds.sh

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Delete service account after testing
+SERVICE_ACCOUNT=$1
+CREDS=$2
+gcloud auth revoke $SERVICE_ACCOUNT
+gcloud iam service-accounts delete $SERVICE_ACCOUNT
+rm $CREDS
```

Comment:
You can rename the file name -> revoke_permission_and_delete_service_account_and_key_file.sh

and CREDS -> KEY_FILE